testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication_collections.py

```diff
@@ -0,0 +1,131 @@
+import uuid
+import pytest
+from CBLClient.Database import Database
+from CBLClient.Collection import Collection
+from CBLClient.Document import Document
+from CBLClient.Replication import Replication
+from libraries.testkit import cluster
+from keywords.ClusterKeywords import ClusterKeywords
+from libraries.testkit.admin import Admin
+from keywords import couchbaseserver
+from keywords.utils import log_info
+from keywords.MobileRestClient import MobileRestClient
+from keywords.constants import RBAC_FULL_ADMIN
+from keywords.SyncGateway import sync_gateway_config_path_for_mode
+from test_replication import verify_sgDocIds_cblDocIds
+
+bucket = "data-bucket"
+
+
+@pytest.fixture
+def teardown_doc_fixture():
+    def _delete_doc_if_exist(sg_client, url, db, doc_id, auth):
+        if sg_client.does_doc_exist(url, db, doc_id) is True:
+            sg_client.delete_doc(url, db, doc_id, auth=auth)
+    yield _delete_doc_if_exist
+
+
+@pytest.fixture(scope="function")
+def scope_collection_test_fixture(params_from_base_test_setup):
+    random_suffix = str(uuid.uuid4())[:8]
+    db_prefix = "db_"
+    scope_prefix = "scope_"
+    collection_prefix = "collection_"
+    cbl_db_name = db_prefix + random_suffix
+    scope = scope_prefix + random_suffix
+    collection = collection_prefix + random_suffix
+    need_sgw_admin_auth = params_from_base_test_setup["need_sgw_admin_auth"]
+    sg_username = "scope_collection_user" + random_suffix
+    log_info("****")
```

Comment:
Do we need these prints?

Reply:
No, this is just debugging prints. Will rectify this.